samples/Kconfig

```diff
@@ -0,0 +1,8 @@
+# General configuration options for samples
+
+# Copyright (c) 2025 The Zephyr Project Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+config SYSTEM_WORKQUEUE_STACK_MIN_SIZE_SAMPLE
+	int "System workqueue stack size for a sample application"
+	default 0
```

Comment:
don't really want to add a Kconfig file for samples - also note that this file is global, it will be present in all applications